.github/workflows/build-luajit.yml

```diff
@@ -16,8 +16,120 @@ jobs:
         ./build_luajit.sh x86_64-linux
 
     - name: Archive results
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
       with:
-        name: Linux luajit
+        name: x86_64-linux-luajit-${{ github.sha }}
         path: |
-          ./share/build/luajit*.tar.gz
+          ${{ github.workspace }}/share/ext/build/luajit*.tar.gz
+
+
+  build-macos:
```

Comment:
I had to add this workflow to dev so that I could then iterate on it in this PR.

Anyway, this seems to work well and it can hopefully be useful also in the future if we wish to use something similar for other builds in the future.

Reply:
Did you check that 32bit bytecode works?

Reply:
Yes! Tested an Android build only armv7 32 bit